internal/kokoro/synth/synth.sh

```diff
@@ -24,7 +24,7 @@ pushd google-api-go-generator
 make all
 popd
 
-if [[ -z "$(git status --porcelain)"]]; then
+if [ -z "$(git status --porcelain)"]; then
```

Comment:
I think we can keep the two brackets, but we need a space between " and ].
```suggestion
if [[ -z "$(git status --porcelain)" ]]; then
```

Reply:
Done